.github/workflows/auto-merge.yml

```diff
@@ -18,12 +18,12 @@ name: auto-merge HEAD to BASE
 on:
   pull_request_target:
     branches:
-      - branch-24.04
+      - branch-24.06
     types: [closed]
 
 env:
-  HEAD: branch-24.04
-  BASE: branch-24.06
+  HEAD: branch-24.06
+  BASE: branch-24.08
```

Comment:
We'll only merge PRs of the repo spark-rapids-jni onto BASE(branch-24.08) from HEAD(branch-24.06), but not merge submodule(rapidsai/cudf) PRs